packages/databricks-vscode/package.json

```diff
@@ -275,6 +275,18 @@
                 "title": "Copy",
                 "enablement": "databricks.context.activated",
                 "category": "Databricks"
+            },
+            {
+                "command": "databricks.bundle.variable.openFile",
+                "title": "Set bundle variables",
```

Comment:
The name can be a bit confusing, I would expect to "set" the varialbes in the `databricks.yml`. Should it be "override"?